collaborations/blood-hunter-2022/bloodhunter-mutant.xml

```diff
@@ -0,0 +1,327 @@
+		</description>
+		<sheet display="false">
+			<description>You gain the Blood Curse of Corrosion for your Blood Maledict feature. This doesn’t count against your number of blood curses known.</description>
+		</sheet>
+		<rules>
+			<grant type="Class Feature" id="ID_DDB_BH22_CLASS_FEATURE_BLOOD_HUNTER_BLOOD_CURSE_OF_CORROSION" />
+		</rules>
+	</element>
+	
+	<element name="Exalted Mutation" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_MUTANT_EXALTED_MUTATION">
+		<description>
+			<p>At 18th level, your body has adapted to produce mutagens naturally in a moment of need. As a bonus action, choose one mutagen currently affecting you. Its effects and side effects end, and you can immediately have a mutagen you know the formula for take effect in its place.</p>
+			<p class="indent">You can use this feature a number of times equal to your Hemocraft modifier (minimum of once). You regain all expended uses when you finish a long rest.</p>
+		</description>
+		<sheet action="Bonus Action" usage="{{exalted mutation:usage}}/Long Rest">
+			<description>You can choose one mutagen currently affecting you to flush from your system and end, then immediately have a mutagen you know the formula for take effect in its place.</description>
+		</sheet>
+		<rules>
+			<stat name="exalted mutation:usage" value="1" bonus="base" />
+			<stat name="exalted mutation:usage" value="hemocraft:modifier" bonus="base" />
+		</rules>
+	</element>
+
+	<!--Mutagens-->
+	<element name="Aether" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_MUTANT_MUTAGEN_AETHER">
+		<supports>Blood Hunter Mutagen 2022</supports>
+		<prerequisite>11th level</prerequisite>
+		<requirements>[level:11]</requirements>
+		<description>
+			<p><em>Prerequisite: 11th level</em></p>
+			<p>You have a flying speed of 20 feet for 1 hour. However, you have disadvantage on Strength checks and Dexterity checks during this time.</p>
+		</description>
+		<sheet>
+			<description>You have a flying speed of 20 feet for 1 hour. However, you have disadvantage on Strength checks and Dexterity checks during this time.</description>
+		</sheet>
+	</element>
+	<element name="Alluring" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_MUTANT_MUTAGEN_ALLURING">
+		<supports>Blood Hunter Mutagen 2022</supports>
+		<description>
+			<p>Your skin and voice become malleable, allowing you to enhance your appearance and presence. You have advantage on Charisma checks. However, you have disadvantage on initiative rolls.</p>
+		</description>
+		<sheet>
+			<description>You have advantage on Charisma checks. However, you have disadvantage on initiative rolls.</description>
+		</sheet>
+	</element>
+	<element name="Celerity" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_MUTANT_MUTAGEN_CELERITY">
```

Comment:
i feel like this feature should actually change the stats but i could be wrong

Reply:
All of the mutagens require a bonus action to actually take effect. They don't need rules added to them.